protocols/candle/index.json

```diff
@@ -0,0 +1,7 @@
+{
+  "cname": "candle",
+  "description": "Candle is the first spiritual, human-focused and inter-connected metaverse protocol.",
+  "path": "candle",
```

Comment:
add `"type": "compoundish" ,`

Reply:
While adding this in gov-sdk , the adapters we would use would be compoundgovernor Alpha or Bravo, test it and see which one works